HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -889,12 +889,6 @@ def check_for_errors
 
       errors += HPXML::check_dates('Daylight Saving', @dst_begin_month, @dst_begin_day, @dst_end_month, @dst_end_day)
 
-      if not @schedules_filepath.nil?
-        unless File.exist? @schedules_filepath
-          errors << "Schedules file path '#{@schedules_filepath}' does not exist."
-        end
-      end
```

Comment:
Had to move this to outside the HPXML class because we want a relative path to be relative to the HPXML file path, but when you are creating an HPXML file from scratch, there is no file path until the HPXML object is written to disk.